src/JsonSchema/SchemaFactory.php

```diff
@@ -110,6 +110,7 @@ public function buildSchema(string $className, string $format = 'json', string $
         }
 
         $definition = new \ArrayObject(['type' => 'object']);
+        $definition['additionalProperties'] = false;
```

Comment:
We should set this flag only if the `allow_extra_attributes` normalization context option is set to `false`: https://github.com/api-platform/core/issues/1217